polygeist/tools/cgeist/Lib/clang-mlir.cc

```diff
@@ -443,6 +437,9 @@ mlir::Value MLIRScanner::createAllocOp(mlir::Type t, VarDecl *name,
     } else {
       mr = mlir::MemRefType::get(1, t, {}, memspace);
       alloc = abuilder.create<mlir::memref::AllocaOp>(varLoc, mr);
+      LLVM_DEBUG(llvm::dbgs() << "MLIRScanner::createAllocOp: created: ";
+                 alloc.dump(); llvm::dbgs() << "\n");
```

Comment:
Please use {} for LLVM_DEBUG region that contains more than one statement.
```suggestion
      LLVM_DEBUG({llvm::dbgs() << "MLIRScanner::createAllocOp: created: ";
                 alloc.dump(); llvm::dbgs() << "\n";});
```

Reply:
done